resolwe/flow/serializers/process.py

```diff
@@ -11,8 +11,26 @@ class Meta:
         """ProcessSerializer Meta options."""
 
         model = Process
-        update_protected_fields = ('contributor', )
-        read_only_fields = ('id', 'created', 'modified')
-        fields = ('slug', 'name', 'data_name', 'version', 'type', 'flow_collection', 'category',
-                  'persistence', 'description', 'input_schema', 'output_schema', 'requirements',
-                  'run', 'scheduling_class') + update_protected_fields + read_only_fields
+        read_only_fields = (
+            'id',
+            'created',
+            'modified',
+        )
+        update_protected_fields = (
+            'name',
```

Comment:
Is there any logic how these are ordered? If not, you can order them alphabetically.

Reply:
No logic, but the order is preserved in the output data. Do we care about the order?